metals/src/main/scala/scala/meta/internal/metals/TastyHandler.scala

```diff
@@ -0,0 +1,184 @@
+package scala.meta.internal.metals
+
+import java.net.URI
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.util.Failure
+import scala.util.Success
+import scala.util.Try
+
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.io.AbsolutePath
+
+import com.google.gson.JsonPrimitive
+import org.eclipse.{lsp4j => l}
+
+// response which is send to the lsp client. Because of java serialization we cannot use
+// sealed hierarchy to model union type of success and error.
+// Moreover, we canot also use Option, so instead every field is nullable
```

Comment:
Maybe let's switch this to valid scaladocs. Acutally same for the `TastyHandler`. Whenever we add new handlers I always try to remember to add scaladocs to them since it's fresh in our mind now, but for a newcomer later on it's really nice to have that info there.

Reply:
```suggestion
// response which is sent to the lsp client. Because of java serialization we cannot use
// a sealed hierarchy to model union type of success and error.
// Moreover, we cannot use Option, so instead every field is nullable
```